build/__phpVersions.sh

```diff
@@ -4,6 +4,12 @@ PHP_RUNTIME_BASE_TAG='20201215.2'
 PHP_FPM_RUNTIME_BASE_TAG='20201215.2'
 COMPOSER_VERSION='1.9.3'
 COMPOSER_SETUP_SHA384='756890a4488ce9024fc62c56153228907f1545c228516cbf63f885e036d37e9a59d27d63f46af1d4d07ee0f76181c7d3'
+COMPOSER109_VERSION='1.9.3'
```

Comment:
`COMPOSER190_VERSION`

Reply:
yeah I thought about it and then I felt it will be weird because COMPOSER_110 will be there that stands for 1.10 .. So I guess I would just do major.minor version

Reply:
as discussion, let's make it COMPOSER1_9_VERSION instead, and others be COMPOSER1_10_VERSION COMPOSER2_0_VERSION major version followed by minor version